src/views/Graph.vue

```diff
@@ -1243,8 +1175,8 @@ export default {
     hashGraph (nodes, edges) {
       // generate a hash for this list of nodes and edges
       return nonCryptoHash(
-        nodes.map(n => n.id).reduce((x, y) => { return x + y }) +
-        (edges || []).map(n => n.id).reduce((x, y) => { return x + y }, 1)
+        nodes.map(n => n.id).reduce((x, y) => x + y) +
+        Array.from(edges.keys()).reduce((x, y) => x + y, 1)
```

Comment:
Turning the map into an array might be a bit expensive.

Reply:
Seeing as originally it was doing an `Array.prototype.map()` followed by an `Array.prototype.reduce()`, it shouldn't be any more expensive as there were 2 loops before. However, newer browsers support `edges.keys().reduce()` directly, so we could do something like
```js
const edgesReducer = (x, y) => x + y
// If browser supports Iterator.prototype.reduce(), use it:
edges.keys().reduce?.(edgesReducer) ?? Array.from(edges.keys()).reduce(edgesReducer)
```
to take advantage of it on the browsers that support it?